akka-stream-tests/src/test/scala/akka/stream/DslFactoriesConsistencySpec.scala

```diff
@@ -60,14 +60,15 @@ class DslFactoriesConsistencySpec extends WordSpec with Matchers {
   val jFlow = classOf[javadsl.Flow[_, _]]
 
   "Java DSL" must provide {
-    "Source" which {
-      "allows creating the same Sources as Scala DSL" in {
-        val sClass = akka.stream.scaladsl.Source.getClass
-        val jClass = akka.stream.javadsl.Source.getClass
-
-        runSpec(sClass, jClass)
-      }
-    }
+    // FIXME must investigate the fromGraph method in javadsl.Source
```

Comment:
Hmm, which one?

```
def from[T](graph: PartialFlowGraph, block: japi.Function[FlowGraphBuilder, UndefinedSink[T]]): Source[T] =
```

? This seems to be OK, delegating to the `apply(PartialFlowGraph)(FlowGraphBuilder => UndefinedSink[T])`. Or I misunderstood the comment